.github/workflows/chromatic.yml

```diff
@@ -40,22 +40,31 @@ jobs:
           exitOnceUploaded: true
           onlyChanged: '!(main)' # only turbosnap on non-main branches
 
-      - name: Chromatic - Non Dependency PR / main
+      - name: Chromatic - DCR Non Dependency PR / main
         uses: chromaui/action@v1
         if: ${{ github.event_name == 'push' || (!contains(github.event.pull_request.labels.*.name, 'dependencies') && github.event.pull_request.user.login != 'dependabot')}}
         with:
-          projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
+          projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN__DOTCOM_RENDERING }}
           token: ${{ secrets.GITHUB_TOKEN }}
           autoAcceptChanges: main
           onlyChanged: true
           untraced: '**/(package**.json|yarn.lock)'
-          workingDir: storybooks
+          workingDir: dotcom-rendering
 
-      - name: Chromatic - Dependency PR
+      - name: Chromatic - DCR Dependency PR
         uses: chromaui/action@v1
         if: ${{ github.event_name == 'pull_request' && (contains(github.event.pull_request.labels.*.name, 'dependencies') || github.event.pull_request.user.login == 'dependabot') }}
         with:
-          projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
+          projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN__DOTCOM_RENDERING }}
           token: ${{ secrets.GITHUB_TOKEN }}
           autoAcceptChanges: main
+          workingDir: dotcom-rendering
+
+      - name: Chromatic - All
+        uses: chromaui/action@v1
+        with:
+          projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
+          token: ${{ secrets.GITHUB_TOKEN }}
           workingDir: storybooks
+          exitOnceUploaded: true
+          onlyChanged: '!(main)' # only turbosnap on non-main branches
```

Comment:
Is this going to run in parallel to the DCR one?

Reply:
yeah, in answer to this and the point you make above we're going to sort the naming once each of these PR's (this and the AR one) are in, I should have mentioned this in the description. So dotcom-rendering will become `dcr-all-storybooks` or something similar and just compose the other storybooks.

I'm just conscious of creating more disruption peoples PR's by changing the name now. If you think thats better to do sooner rather than later i can do it now.

The end result should be:

dcr-storybooks-all 
-- dotcom-rendering
-- apps-rendering
-- common-rendering

Reply:
Also we can do some name changing, the underscores were just to differentiate temporarily from the existing names